src/lib/Scenes/Home/Components/Sales/Components/LotsByFollowedArtists.tsx

```diff
@@ -1,91 +1,32 @@
 import GenericGrid from "lib/Components/ArtworkGrids/GenericGrid"
+import Spinner from "lib/Components/Spinner"
+import { isCloseToBottom } from "lib/utils/isCloseToBottom"
+import { get, once } from "lodash"
 import React, { Component } from "react"
-import createEnvironment from "lib/relay/createEnvironment"
-import renderWithLoadProgress from "lib/utils/renderWithLoadProgress"
-import styled from "styled-components/native"
-import { Button, ScrollView, Text, View } from "react-native"
+import { FlatList, ScrollView, Text, View } from "react-native"
 import { RelayPaginationProp } from "react-relay"
+import styled from "styled-components/native"
 import { SectionHeader } from "./SectionHeader"
-import { createPaginationContainer, graphql, QueryRenderer, QueryRendererProps } from "react-relay"
-import { get, once } from "lodash"
-import { isCloseToBottom } from "lib/utils/isCloseToBottom"
 
 const DEFAULT_TITLE = "Lots by Artists You Follow"
-const PAGE_SIZE = 8 // FIXME: Increase
-
-interface RelayProps {
-  title?: string
-  relay: RelayPaginationProp
-  viewer: {
-    sale_artworks: {
-      pageInfo: object
-      edges: Array<{
-        node: object
-      }>
-    }
-  }
-}
+export const PAGE_SIZE = 10
 
 const Container = styled.View`padding: 10px;`
 
-const Query = graphql.experimental`
-  query LotsByFollowedArtistsQuery($count: Int!, $cursor: String) {
-    viewer {
-      ...LotsByFollowedArtists_viewer @arguments(count: $count, cursor: $cursor)
-    }
-  }
-`
-
-const Pagination = createPaginationContainer(
```

Comment:
@damassi To be clear, you could've still used a pagination container here. The issue was that we were creating two query renderers to render the same view. One in [LotsByFollowedArtists](https://github.com/artsy/emission/pull/862/files#diff-de60aba6af766c37ea0f39eaca08d720L73) and one in the  [Sales container](https://github.com/artsy/emission/pull/862/files#diff-a08fc595f15caaf7c0686f93c2fcce70L70). I'm not sure how you're doing the pagination now but if it complicates the implementation, I'd suggest bringing that container back

Reply:
Sorry I just re-read your comment, I guess you're saying we can't have a pagination container inside of a fragment container. If moving the pagination container out of `LotsByFollowedArtists` works then that's fine too

Reply:
We can definitely have whichever type of container be children of any other type of container, so something else must be going on.

Wanna pair on that?

Reply:
Yeah it was weird! whenever it existed down the chain the data would come back `undefined`. I tried all sorts of things but it wasn't until I moved everything into one spot that it would work. I'm gonna loop back around to the remaining Auctions issues but will add a note to follow up on this next week; def want to pair and see it with my own eyes. If composition is flexible across children with different data containers then this aint so bad :)